### PR TITLE
v2.1.0

### PR DESCRIPTION
**Pull request by quantum9Innovation**
# Merge v2.1.0 from `nightly` into `master`
___

## New Feature
Introducing pre-compiled models!
Compile a common model with just one function call to `epispot.pre`.
## Known Issues
Code coverage is now @ 98% :fireworks:
However, a few new errors were discovered in tests; see
 - #20 
 - #21 
## Code Breakdown
Merged the `epispot/` and `tests/` directories from `nightly` into `v2.1.0`
Ex:
 - epispot/pre.py
    - Use this to define pre-compiled models!
  - epispot/*
     - Marked code with `# pragma: no cover` to indicate exemption from code coverage
 - tests/CI/
    - Added new tests for `pre.py`
    - Added new tests to increase code coverage
 - .github/workflows/build.yml
    - Trigger code coverage on change to main package
    - Added new tests

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
LGTM alerts will be fixed in a future release: **ignore for now**

**Comment by quantum9Innovation** [comment]
Waiting on CodeQL and LGTM (if there are any new issues they will be fixed in a later release)

**Comment by quantum9Innovation** [comment]
@Quantalabs ready to merge after review

File: .github/workflows/coverage.yml
Comment (Quantalabs) [inline .github/workflows/coverage.yml:6]: There is no "codecov" branch. Change this to "main" if you are merging this into main.

**Review by Quantalabs — CHANGES_REQUESTED** [review]
Comment on `coverage.yml` workflow.

**Pull request merged**
